pkg/api/v1alpha1/cluster.go

```diff
@@ -740,3 +759,39 @@ func validatePodIAMConfig(clusterConfig *Cluster) error {
 	}
 	return nil
 }
+
+func validateCPUpgradeRolloutStrategy(clusterConfig *Cluster) error {
+	if clusterConfig.Spec.ControlPlaneConfiguration.UpgradeRolloutStrategy == nil {
+		fmt.Println("ControlPlaneConfiguration: UpgradeRolloutStrategy not specified in cluster config. CAPI will default to 'RollingUpdate' with maxSurge=1")
```

Comment:
Use logger instead of `fmt.Println`
```suggestion
		logger.Info("ControlPlaneConfiguration: UpgradeRolloutStrategy not specified in cluster config. CAPI will default to 'RollingUpdate' with maxSurge=1")
```